### PR TITLE
test: refactor cp tests to ava

### PR DESCRIPTION
**Pull request by nfischer**

### CODE REVIEW

File: ava-test/cp.js [hunk 1]
```diff
@@ -0,0 +1,651 @@
+import fs from 'fs';
+
+import test from 'ava';
+
+import shell from '..';
+import common from '../src/common';
+import utils from './utils/utils';
+
+const numLines = utils.numLines;
+const skipOnWinForEPERM = require('./utils/utils').skipOnWinForEPERM;
```
Comment (freitagbr) [inline ava-test/cp.js]:
```javascript
const { numLines } = utils;
const { skipOnWinForEPERM } = utils;
```

Comment (nfischer) [inline ava-test/cp.js]: I changed it all to `utils.numLines()` and the like.

**Comment by nfischer** [comment]
@freitagbr PTAL

**Comment by freitagbr** [comment]
LGTM

**Comment by nfischer** [comment]
Merging, since I think lgtm.co is broken...

**Pull request merged**